### ISSUE TITLE
Onboarding flow for new user

### ISSUE THREAD

**Issue by OAyomide** [agbowo/agbowo-app#2]
Right now, the onboarding assumes the user uses Gtbank. Also, the greeting label is currently hardcoded with my name (not standard practice I agree and besides its for initial design/implementation).

It'll be nice to have a standard way to onboard users. Since the App doesnt use account creation (we dont need that. I dont see use needed it anytime soon if at all), we should have a way to personalize the experience for the user --Name, avatar, etc.

Part of the standard onboarding can include: 

- Name of the user (or alias)
- Bank (and account. since not all banks put the full account number in the SMS alert)
- Icon/Pix (like a standard app out there)

**Comment by OAyomide** [comment]
@Itope84 @Lord-Jerry RFC/thoughts?

**Comment by Lord-Jerry** [comment]
we could ask users to fill in a name they would like to be called.

**Comment by OAyomide** [comment]
I think the next step now is the design. Opened #3 . @Lord-Jerry anything else you think we can include in the onboarding screen?

**Comment by Lord-Jerry** [comment]
not sure we need anything else on the onboarding.

**Comment by Itope84** [comment]
Also, I should mention that while setting up, I noticed that the screen where you select bank, doesn't render properly on iOS.

Should we worry about iOS?

**Comment by OAyomide** [comment]
Yeah of course we should! but i dont use an iphone... or have a macbook. So if we can get Ios help, it'd be great.

But we really should.

**Comment by Itope84** [comment]
Okay then, if I see anything that doesn't work fine on iOS, I'll make PRs for them 🙌
